### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by cheops**
first example of check_rate_limit
if this is ok, I will make more for the other functions

the async example is fully async, also for reading the config file, but that requires the additional library aiofiles
should a comment about that be added?
I added a small readme.md to the examples folder, feel free to adapt it after you merge.

### CODE REVIEW

File: test_live_async.py [hunk 1]
```diff
@@ -110,6 +110,28 @@ async def test_deletestatus(config):
         assert await delete_result.text() == "OK 200: Deleted Status"
 
 
+async def test_addoutput(config):
```
Comment (cheops) [inline test_live_async.py:113]: can you test this after merging?

File: test_live_sync.py [hunk 2]
```diff
@@ -72,6 +72,21 @@ def test_deletestatus(pvo):
     assert delete_result.text == "OK 200: Deleted Status"
 
 
+def test_addoutput(pvo):
```
Comment (cheops) [inline test_live_sync.py:75]: can you test this after merging?

**Comment by cheops** [comment]
I think only the last 2 commits d08b43b and f694bb2 are not merged yet.
Sorry for the mess, I hoped that commits would be better tracked across repositories when they were already merged.

File: test_live_sync.py (hunk 2 above)
Comment (yaleman) [reply test_live_sync.py:75]: I'll try and set up a test for it with a test system, I don't want to mess up my data 😄

**Pull request merged**